src/vue/backend/src/store/modules/general/actions.js

```diff
@@ -7,6 +7,13 @@ const generalSettingsStateChange = ({ commit }, payload) => {
     })
 }
 
+const apiCall = () => {
```

Comment:
We will need this to be more specific because there will be multiple api calls in this file. Try something like `verifyAPIkey`